src/frameworks/next/utils.ts

```diff
@@ -245,7 +245,7 @@ export async function isUsingNextImageInAppDirectory(
   projectDir: string,
   nextDir: string
 ): Promise<boolean> {
-  const { found: files } = new Glob(
+  const files = globSync(
```

Comment:
I think this should work fine - however, we probaly dont need the `await` on l252 any more.

Can you remove that and then test this again?

Reply:
done, thanks! Just added an image test to the deploy tests too, I'll check if everything goes well